MUXSDKKaltura/MUXSDKKaltura/MUXSDKPlayerBinding.swift

```diff
@@ -383,7 +376,7 @@ public class MUXSDKPlayerBinding: NSObject {
         
         eventVideoData.videoSourceUrl = videoData.url
         
-        if (self.videoData.lastAdvertisedBitrate > 0) {
+        if (self.videoData.lastAdvertisedBitrate > 0 && self.videoData.started) {
```

Comment:
Doesn't this only send the bitrate to the server if the video has started playing? But afaik we detect the first rendition before playback starts - so we need to make sure the bitrate is still sent for that first rendition as part of the metrics in the video data. We only want to dispatch an actual `renditionchange` event after playback starts.

or am I misreading this and we will eventually send the first rendition's bitrate but it will be after the video starts?